src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/SingleShot.cs

```diff
@@ -79,10 +81,19 @@ public void Dispose()
         public TensorsData Invoke(TensorsData inTensorsData)
         {
             TensorsData out_data;
-            IntPtr out_ptr;
+            IntPtr out_ptr = IntPtr.Zero;
             NNStreamerError ret = NNStreamerError.None;
 
-            ret = Interop.SingleShot.InvokeSingle(_handle, inTensorsData.Handle, out out_ptr);
+            NNStreamer.CheckNNStreamerSupport();
+
+            try
+            {
+                ret = Interop.SingleShot.InvokeSingle(_handle, inTensorsData.Handle, out out_ptr);
+            }
+            catch (DllNotFoundException e)
```

Comment:
Unnecessary check because if pass the CheckNNStreamerSupport(), There is no case of DllNotFoundException.

Reply:
ACK!